Gemfile.lock

```diff
@@ -65,20 +65,21 @@ GEM
     diff-lcs (1.1.3)
     equalizer (0.0.11)
     erubis (2.7.0)
-    ethon (0.9.1)
+    ethon (0.10.1)
```

Comment:
> Is there a better way that doesn't result in the versions changing?

`bundle install` should not update the Gemfile.lock (`bundle update` would though).

In any case, assuming you have all dependencies installed (after running `bundle install`), you can `git add -p` the relevant hunk for PostgreSQL and `git checkout` the file.

There is usually no issue running slightly different versions (i.e. it shouldn't impact your testing), but for the official version / build, we like to be prudent and keep a dependency tree that we have tested ourselves.